### ISSUE TITLE
Failure on voiceOver.start if VoiceOver is already started

### ISSUE THREAD

**Issue by cssinate** [guidepup/guidepup#100]
Hello! First let me say - I love this initiative. Thank you for investing your time in accessibility testing. I'm really excited about this project.

It seems `voiceOver.start()` is automatically started in our Playwright tests. I couldn't figure out how/where that's happening. But it seems to me if `voiceOver.start()` is called and VoiceOver is already running that it shouldn't throw an error.

Here's my issue: I want to ensure that even if a test fails that `voiceOver.stop()` is called. So I wanted to do a `beforeEach` and `afterEach` to start and stop voiceOver. When `start()` is called, it throws an error.

Is there a better way to do this, or a way around it? Might you consider a way to bypass the error? Or is there a way I can't see to check if VoiceOver has started before I run the `start()` function?

Obviously, I can `try { voiceOver.start() } catch (e) { // check that error message is "already running" string }` but that feels a little messy to me.
